stats/src/test/java/io/airlift/stats/TestGarbageCollectionNotificationInfo.java

```diff
@@ -0,0 +1,50 @@
+package io.airlift.stats;
+
+import com.sun.management.GcInfo;
+import org.testng.annotations.Test;
+import sun.management.GarbageCollectionNotifInfoCompositeData;
+
+import java.lang.management.GarbageCollectorMXBean;
+import java.lang.management.ManagementFactory;
+
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+
+public class TestGarbageCollectionNotificationInfo
```

Comment:
Since this PR is a simple refactor, I'm not sure if this test really has to be added.